mantle/lattice/mantle40/decode.py

```diff
@@ -18,7 +18,7 @@ def Decode(i, n, invert=False, **kwargs):
     @return: 1 if the n-bit input equals i
     """
 
-    assert n <= 8
+    #assert n <= 8
```

Comment:
Looks good, I reviewed the implementation and it seems to default to using `EQ` to implement decode. I think the assertion was left over when it used just the specialized `LUT` implementation. Can you remove this line completely rather than just commenting it out?